[CONTRIBUTING.md] - quick update re: triggering i18n or not (#1079)
withastro/docs: CONTRIBUTING.md

Contributions to the documentation site are made by editing the docs repository. You can do this directly on GitHub.com or by creating a copy of the repository locally, making your changes there, and contributing back to our repository.

> > By default, your merged PR to an English page will trigger our Translation Status Tracker. If your change should **NOT** be applied to every language (e.g. a typo fix to an English word), please include the word "minor" in your PR title and/or in a commit message.  See the next section for more details.

Reviewer: Do we want to avoid using `<blockquote>` for things that aren’t quotations?

If we do want to highlight this I guess we could use the GitHub syntax I suggested in #1078 here too:

```suggestion
> **Note**
> By default, your merged PR to an English page will trigger our Translation Status Tracker. If your change should **NOT** be applied to every language (e.g. a typo fix to an English word), please include the word "minor" in your PR title and/or in a commit message.  See the next section for more details.
```

I _think_ GitHub unfortunately still uses `<blockquote>` to render these, but given they’re detecting the custom syntax, I suppose there’s hope they might switch to better markup in the future.

Author: Yes, we *do* want to avoid `<blockquote>` -- I searched and couldn't find GitHub's new "note" notation, that I thought was of the form `:::note ` like ours, but it didn't preview well here. Does anyone know?

Reviewer: Oh! It’s my suggestion! Here’s the page with details: https://github.com/github-community/community/discussions/16925

> > By default, your merged PR to an English page will trigger our Translation Status Tracker. If your change should **NOT** be applied to every language (e.g. a typo fix to an English word), please include the word "minor" in your PR title and/or in a commit message.  See the next section for more details.

Reviewer 2: I think only the squash merge commit (that generally is the title) is used to mark the PR as minor/major. If this is really the case, we need to change the wording a bit to not imply that you can select which pages changed are considered minor/major changes in the same PR.

Author: Ah, I hadn't considered people might think that only certain commits would trigger. 

So maybe, although it's really coming from commit message, I'll just be sneaky here and say PR title for simplicity.

Contributions to the documentation site are made by editing the docs repository. You can do this directly on GitHub.com or by creating a copy of the repository locally, making your changes there, and contributing back to our repository.

> **Note**

> > By default, your merged PR to an English page will trigger our Translation Status Tracker. If your change should **NOT** be applied to every language (e.g. a typo fix to an English word), please include the word "minor" in your PR title and/or in a commit message.  See the next section for more details.

Author: ```suggestion
> By default, your merged PR to an English page will trigger our Translation Status Tracker. If your change should **NOT** be applied to every language (e.g. a typo fix to an English word), please include the word "minor" in your PR title.  See the next section for more details.
```

I'm changing to this, but I'm making a note that after using this for a while, people are often making "minor" PRs that are small, but *should* trigger i18n updates. I'm thinking that we were being too clever with this word choice, and might prefer we think of a word that is actually related to needing a translation or not.

**Internationalization (i18n)**

Please only add new text content to the docs **in English**, by modifying only **`.md` files located within `src/pages/en/`**. 

We have automated systems in place for notifying our community translators that there is new material to be translated, so there is no need to make changes to additional languages yourself. 

> Our Docs are translated into several languages and we rely on automation to notify our translators that English pages have changed. By default, when a PR to an English page is merged, our Translation Status Tracker is updated.  **Please consider whether your PR should or should NOT trigger a rewrite to pages in other languages**: some tiny fixes are English only (e.g. spelling of English words) but some *will* require updating all language pages e.g. a small error in a code sample.  Please use the word "minor" in  your PR title and/or in a commit message to indicated that your PR does **NOT** require translating.

Author: ```suggestion
Our Docs are translated into several languages and we rely on automation to notify our translators that English pages have changed. By default, when a PR to an English page is merged, our Translation Status Tracker is updated.  

**When choosing a PR title, please consider whether your PR should or should NOT trigger a rewrite to pages in other languages**: some tiny fixes are English only (e.g. spelling of English words) but some *will* require updating all language pages (e.g. a small error in a code sample).  Please use the word "minor" in  your PR title to override this default behavior and indicate that your PR does **NOT** require translating.
```
